ops/aws/canary/lambda/pkg/scenarios/list.go

```diff
@@ -2,15 +2,14 @@ package scenarios
 
 import (
 	"context"
-	"github.com/rs/zerolog/log"
 
-	"github.com/filecoin-project/bacalhau/cmd/bacalhau"
+	"github.com/rs/zerolog/log"
 )
 
 func List(ctx context.Context) error {
 	// intentionally delay creation of the client so a new client is created for each
 	// scenario to mimic the behavior of bacalhau cli.
-	client := bacalhau.GetAPIClient()
+	client := getClient()
```

Comment:
`bacalhau.GetAPIClient()` would set `apiHost, apiPort` to production values.
`getClient()` will use test values if applicable